PyInstaller/exceptions.py

```diff
@@ -11,4 +11,11 @@
 
 
 class ExecCommandFailed(SystemExit):
-    pass
+    def __init__(self, module_name, hook_module_name):
+        self.module_name = module_name
+        self.hook_module_name = hook_module_name
+    def hook_import_fail(self):
+        return f"""Failed to import module {self.module_name} required by hook for module {self.hook_module_name}.
```

Comment:
This is the reason the CI is failing for Python 3.5 is because of this - `f"strings"` were introduced in 3.6.

Reply:
Further: Lines are much too long and the consecutive lines of multi-line string will printed with many spaces in front.

Reply:
>>> print("""Failed to import module {self.module_name} required by hook for module {self.hook_module_name}.
...                 Please check whether module {self.module_name} actually exists and whether the hook is
...                 compatible with your version of {self.hook_module_name}: You might want to read more about
...                 hooks in the manual and provide a pull-request to improve PyInstaller.""")
Failed to import module {self.module_name} required by hook for module {self.hook_module_name}.
                Please check whether module {self.module_name} actually exists and whether the hook is
                compatible with your version of {self.hook_module_name}: You might want to read more about
                hooks in the manual and provide a pull-request to improve PyInstaller.